client/src/graphql/mock.ts

```diff
@@ -0,0 +1,71 @@
+import { makeUniqueId } from "@apollo/client/utilities";
```

Comment:
When this mock file gets extended, I didn't want dev-created id to have duplicates somewhere down the line. Since `id` is expected to be a UUID, using this, especially since these are for testing.